sys/sysmon/PB_MemoryStatus.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+// All message type names are prefixed with "PB_" to avoid collisions
+// C++ classes already defined in code and Protobuf generated types.
+
+message PB_MemoryStatus {
+    uint32 memory_size = 1;
+    uint32 memory_free = 2;
+    uint32 memory_used = 3;
```

Comment:
Is size == free + used?
If yes, you could remove one of those variables since it can be computed, and this will reduce the message size.

Reply:
done

Reply:
There is still `memory_free` here.

Reply:
done